CONTRIBUTING.md

```diff
@@ -51,13 +53,13 @@ If you want to run the end-to-end tests locally you'll need [Docker](https://www
 
 ## Testing
 
-A full guide to testing can be found in the [testing](./TESTING.md) document
+A full guide to testing can be found in the [testing](./docs/TESTING.md) document
 
-## CI
+### CI
 
 CI runs on Buildkite. Tests are run automatically on any branch from within this repo. PRs from external repos do not run on the private test infrastructure. Once an external PR has been reviewed by a Bugsnag employee, a branch can be created within this repo in order to run on CI.
 
-⚠️ __Caution__: exercise due-diligence before creating a branch based on an external contribution – for example, be sure not to merge a bitcoin miner disguised as a bug fix!
+⚠️ __Caution__: exercise due-diligence before creating a branch based on an external contribution
 
 ## Keeping dependencies in sync
```

Comment:
How about making a section called "Supporting new Expo SDK versions" after "Adding new features" to contain all the bits around bumping the SDK?

We've also recently added an example, so we should consider what our approach for this should be – just keep the latest or have one per supported version, deleting the older ones. This should also be included in this section.